decoy/mypy/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""Decoy mypy plugin entrypoint."""
```

Comment:
This change required because mypy is unable to understand how to use a local plugin file called `mypy.py` (seems to be a bug in mypy's plugin loading system)